RecoEgamma/EgammaIsolationAlgos/python/egammaHBHERecHitThreshold_cff.py

```diff
@@ -7,11 +7,11 @@
     recHitEThresholdHB = _thresholdsHBphase1,
     recHitEThresholdHE = _thresholdsHEphase1,
     maxHcalRecHitSeverity = cms.int32(9),
+    usePFThresholdsFromDB = cms.bool(False)
 )
 
-egammaHBHERecHit_2023 = egammaHBHERecHit.clone(
-    recHitEThresholdHB = _thresholdsHBphase1_2023
-)
+from Configuration.Eras.Modifier_hcalPfCutsFromDB_cff import hcalPfCutsFromDB
+hcalPfCutsFromDB.toModify(egammaHBHERecHit,
+                   usePFThresholdsFromDB = True)
+
 
-from Configuration.Eras.Modifier_run3_egamma_2023_cff import run3_egamma_2023
-run3_egamma_2023.toReplaceWith(egammaHBHERecHit,egammaHBHERecHit_2023)
```

Comment:
I think it's not a good idea to remove these. If someone wants to run on 2023 samples with `usePFThresholdsFromDB = cms.bool(False)`, then they will get wrong thresholds of HCal, as these lines are deleted. I suggest to keep it for backward compatibility reasons.